srml/democracy/src/lib.rs

```diff
@@ -318,6 +317,28 @@ decl_event!(
 
 decl_module! {
 	pub struct Module<T: Trait> for enum Call where origin: T::Origin {
+		/// The minimum period of locking and the period between a proposal being approved and enacted.
```

Comment:
looks like we're duplicating every const definition. was this intended?! not cool.

Reply:
ahh; i see. this is instead of the `decl_properties` macro...